libraries/core-react/src/Table/Table.tsx

```diff
@@ -8,7 +7,9 @@ const TableBase = styled.table`
   border-collapse: collapse;
 `
 
-export const Table = (props) => {
+type Props = React.HTMLAttributes<HTMLTableElement>
+
+export const Table: FunctionComponent<Props> = (props: Props) => {
```

Comment:
Should we use `React.FC` instead of `FunctionComponent` since we’re using `React.HTMLAttributes`? 🤔

Reply:
…just a thought. But if the fettblog author don’t recommend using shorthand I’m not going to argue against that 😛

Reply:
Might be? I'm actually not sure about what I think about `React.HTMLAttributes`... You would after all write 
```
import React, {useState} from 'react
...
const t = useState(...)
```
instead of React.useState right? So I kind of feel it's quite verbose to repeat React ?

Reply:
True. So then maybe we should change that instead, and `import { HTMLAttributes }`.

Reply:
What is you just sett the return type to `JSX:Element` ? 

Repeating "Props" in generic and parameters seems messy

Reply:
Maybe? I think the current code is more the result of copy'n paste from different sources than consistency

Reply:
I don't actually feel we are repeating more than usual? But could write it like this instead
```
type Props = React.HTMLAttributes<HTMLTableElement>

export const Table: FunctionComponent<Props> = ({children) => {
  return <TableBase {...props}>{children}</TableBase>
}
```

Reply:
On second thought, I dont think you need to set the type on the parameter when you are setting the type on variable declaration @wenche

Reply:
And yeah, you can do it like that @mimarz 
```
type Props = React.HTMLAttributes<HTMLTableElement>

export const Table = (props: Props): JSX.Element => {
  const { children } = props
  return <TableBase {...props}>{children}</TableBase>
}
```

Reply:
Maybe just tune that Eslint rule? 
https://www.reddit.com/r/reactjs/comments/i4jx85/reactfc_vs_jsxelement/